### ISSUE TITLE
Requiring an ORCID iD from users who apply for credentialed access

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/physionet-build#1282]
An ORCID iD is a free, unique, persistent identifier that can be used to distinguish one researcher from another. The iD can be connected to professional information—affiliations, grants, publications, peer review, etc. https://info.orcid.org/benefits-for-researchers/

We currently recommend that applicants link an ORCID iD to their account before applying for credentialed access. In recent meetings, there has been some support for making this a requirement. 

I'm not aware to any barriers that would prevent someone from creating an ORCID iD, but to be on the safe side we can provide a contact email address so that applicants can reach out to us if they have any issues linking an ORCID iD.

@bemoody @alistairewj @kepaik @rgmark @briangow @Lucas-Mc @li-lcp @lceli @kepaik please add your thoughts on requiring, rather than recommending, ORCID iDs for users applying for credentialed access!

**Comment by briangow** [comment]
It's possible that some people, especially students who haven't published previously, will create and ORCID iD but their account won't contain any useful information.  They would still be able to link to their ORCID account through PhysioNet but in this case we might not learn anything about them.  Do we want to require specific information within their ORCID profile?  If they are a student and don't have any publications what would we require for information?  The data fields for each section in an ORCID profile are listed here: https://support.orcid.org/hc/en-us/articles/360006894714-Metadata-in-the-Education-and-Qualification-section (click each section on the left).

**Comment by tompollard** [comment]
@briangow good idea, it would be great if we could encourage users to populate their profiles. 

It looks like it is possible for users to link their ORCID accounts to institutional accounts/logins too. Need to look into this a little more, but perhaps it is also something that we could recommend. 

A side benefit of users linking their ORCID iDs to PhysioNet is that we can populate their ORCID profiles with data/software resources that they publish.

**Comment by kepaik** [comment]
Leo's email comments: 
> I suggest asking the referee's (instead of the applicant's) ORCID number. Two alternatives if the referee does not have an ORCID number are:
> 1. the referee's Google Scholar page
> 2. the referee's profile page on the company website
> In addition, we need the referee's institution/company email address that we can email to verify the applicant's identity.
> I think these are small asks in exchange for access to our resources.

- The small issue having the student provide the referee's ORCID iD is that cuts out one of the security protocols of the ORCID ID that we would use, where the individual logs in and confirms the identity, otherwise it's equivalent to sending a webpage link.

- Brian's point is a valid concern, since it seems anyone can create an ORCID iD... so just having one doesn't necessarily improve the ability to confirm their identity. It only proves that they were willing to go through and confirm their email information.  It only provides security if they have linked publications or are associated with an NIH grant. Because of this, I don't think we could ever solely rely on the ORCID iD to establish identity.

It might be helpful to talk about the overarching objective, rather than the specific tool (ORCID iD). We could delineate this as two major objectives: 
1. Establishing the applicant's identity
2. Establishing the reference's identity

And then using a combination of tools to do so. Obviously if the applicant has a well fleshed out ORCID iD, that allows us to shortcut approval. But if they don't, the other factors such as organizational email and provided weblinks (Google scholar/pubmed) would help.  I guess a followup question is how do we handle (or if we want to) applicants that can't meet these requirements.

**Comment by briangow** [comment]
Currently the ORCID iD for a reference / referee works similarly to that 
for an applicant.  For this to be displayed within credentialing, the 
reference has to link to their ORCID account under their PhysioNet 
profile.  If the applicant supplies the email of a reference who has 
linked to ORCID in this manner, it will be displayed.

Brian

**Comment by tompollard** [comment]
For discussion about ORCID iDs for referees, also see this thread: https://github.com/MIT-LCP/physionet-build/issues/1283

**Comment by Lucas-Mc** [comment]
Requiring it is fine but we shouldn't require their ORCID profile to be populated with anything. That's adds burden for everyone when we say "Reject: please fill out X on your ORCID profile" when it was supposed to be a purely beneficial thing.

I keep trying to think of any exclusion bias we would be committing by requiring ORCID but I can't think of any. I understand we think it's pretty standard now but we are from the United States and publish in high impact factor journals so not exactly the best point of view... Anyway, even if some countries or low impact factor journals haven't adopted it yet I guess the user just has to make one (as in my case and it only took a minute).

**Comment by alistairewj** [comment]
It would be great to require ORCID. I believe there is the possibility of verifying institution via ORCID since they support SSO with the eduGAIN folks (if it's not possible now, it's on ORCID's roadmap). That would greatly speed up identity verification for a number of users.

**Comment by briangow** [comment]
Very interesting @alistairewj! I found this link which explains this further: https://info.orcid.org/documentation/integration-guide/sign-into-orcid-with-institutional-credentials/#easy-faq-3777. I was able to sign into ORCID with my MIT account (see the Access through your institution option at the bottom of the ORCID sign in page). Once I link my institution account it shows up under my Account Settings as an Alternate sign in account.  I'm not certain whether we can use the ORCID API to check to see whether an individual has linked to an university account in this manner but the API does allow reading /email and /external-identifiers.  The latter may apply to linking these other (non-university) accounts: https://support.orcid.org/hc/en-us/articles/360006894854-Add-person-identifiers-other-identifiers-to-your-ORCID-record .  

If we decide to require a linked ORCID iD for the applicants themselves, we could require that their ORCID profile either contains information about their publications or a link to a university email (for students who don't have publications yet).  We could use the API to confirm one of these exist in their account.  We may also need to consider cases where the applicant is no longer in school (and perhaps doesn't have access to an alumni university account), and hasn't published previously, but is now starting to do research.  I assume this scenario doesn't apply to many people though.

**Comment by briangow** [comment]
In following up with ORCID, they indicated that:

- There is no way to determine whether an ORCID user has authenticated an institutional account for logging into ORCID.  Here are some of their comments around this.
  - "The ability for users to log in to ORCID using their institutional credentials is really just a convenience for the user and is not part of the ORCID API. If you wanted a way to check to see who is logging in with their institutional credentials, you would only be able to check to see if MIT researchers had signed in to ORCID using their institutional credentials and to do that you would have to look at the MIT IdP system logs on your side, which will probably require working with central IT or whomever manages those logs at MIT."
  - "Unfortunately, there is no way to check this for MIT users or non-MIT users using the ORCID API. When you have the ORCID iDs for each user, you could do an API call to read their email address(es) but that likely will not be helpful since email addresses are set to “private” visibility by default, and a user would need to actively go in to their ORCID record and switch the setting to either trusted parties or public visibility in order for you to read that information. Most users do not do this."
  - "A potential work-around could be to add a step to the log -in or account-creation workflow aside from the ORCID OAuth process, where you could ask each user to enter their institutional email address separately if they have one, as an additional optional layer of confirmation."
- I also followed-up with ORCID to ask if we could force a user to login by using an institutional account but this is not possible.  Here are some of their comments around this.
  - "Indeed, the sign-in options cannot be hidden."
  - "you can emphasize the institutional credential sign-in option in your UI."
